src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] arrayOfFigures = new Figure[6];
+        for (int i = 0; i < arrayOfFigures.length; i++) {
+            arrayOfFigures[i] = (i < 3) ? figureSupplier.getRandomFigure()
+                    : figureSupplier.getDefaultFigure();
+        }
+        for (Figure figure : arrayOfFigures) {
+            figure.draw();
+        }
```

Comment:
You can do it with one for-loop. Imagine 100.000 figures. And now instead of 100.000 iterations, you have 200.000